ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuite.scala

```diff
@@ -91,7 +91,7 @@ private[testtool] abstract class LedgerTestSuite(val session: LedgerSession) {
     }
     assert(actualCode == expectedCode, s"Expected code [$expectedCode], but got [$actualCode].")
     assert(
-      message.contains(pattern),
+      Option(message).getOrElse("").contains(pattern),
```

Comment:
Is the output of `getDescription` nullable?

Reply:
Apparently yes. `ledger.getPackage` returns a `NOT_FOUND` status code with a `null` description.

Reply:
Cool, thanks.